### PR TITLE
Search page

### PR DESCRIPTION
**Pull request by ebraheem002**
search page done !

### CODE REVIEW

File: src/components/IntroSection/IntroSection.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap');
```
Comment (MustafaNawzat) [inline src/components/IntroSection/IntroSection.css:1]: why is this file here?

File: src/components/SearchPage/SearchResult/SearchResult.js [hunk 2]
```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import { Container, Row, Col, Button, Card } from 'react-bootstrap';
+import Avatar from './images/avatar.png';
+import './style.css';
+function SearchResult() {
+  return (
+    <div className="SearchResult">
```
Comment (MustafaNawzat) [inline src/components/SearchPage/SearchResult/SearchResult.js:7]: SearchResult >> searchResult

File: src/components/SearchPage/SearchResult/SearchResult.js [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+      <Container>
+        <Row>
+          <Col>
+            <h3>Search result</h3>
+          </Col>
+
+          <hr className="solid" />
+        </Row>
+        <Row>
+          <Col md={4}>
```
Comment (MustafaNawzat) [inline src/components/SearchPage/SearchResult/SearchResult.js:17]: Don't copy and paste yourself.

- make an array of objects (which each object represent that doctor info)
- then map over them.
this will make you write the Card only one time 

see this ex: https://codesandbox.io/s/compassionate-christian-945lt?file=/src/App.js

File: src/components/IntroSection/IntroSection.css (hunk 1 above)
Comment (ebraheem002) [reply src/components/IntroSection/IntroSection.css:1]: I don't know what is this
